sda/internal/database/db_functions.go

```diff
@@ -684,7 +685,7 @@ func (dbs *SDAdb) GetCorrID(user, path string) (string, error) {
 func (dbs *SDAdb) getCorrID(user, path string) (string, error) {
 	dbs.checkAndReconnectIfNeeded()
 	db := dbs.DB
-	const query = "SELECT DISTINCT correlation_id FROM sda.file_event_log e RIGHT JOIN sda.files f ON e.file_id = f.id WHERE f.submission_file_path = $1 and f.submission_user = $2 AND NOT EXISTS (SELECT file_id FROM sda.file_dataset WHERE file_id = f.id)"
+	const query = "SELECT DISTINCT correlation_id FROM sda.file_event_log e RIGHT JOIN sda.files f ON e.file_id = f.id WHERE f.submission_file_path = $1 and f.submission_user = $2"
```

Comment:
I just want to verify, since this function is also called from other places, eg ingest... wasn't the removed part there for a reason..?

Reply:
Since we don't want two functions that does more or less the same thing with almost the same name, I guess we need to expand the logic to handle the conflicting situations that can arise.

Reply:
Fixed in a better way in the last commit